util/perl/TLSProxy/Message.pm

```diff
@@ -14,6 +14,7 @@ use TLSProxy::Alert;
 
 use constant DTLS_MESSAGE_HEADER_LENGTH => 12;
 use constant TLS_MESSAGE_HEADER_LENGTH => 4;
+use constant RECORD_HANDSHAKE_HEADER_LENGTH => 12;
```

Comment:
I'm confused:
Is this not just DTLS_MESSAGE_HEADER_LENGTH? The DTLS record header length is 13 (for non unified headers at least).

Reply:
This was the Record Header. maybe I made an incorrect Diagnosis. I was seeing the issue with client hello and the certificate message.  If you look at the util/perl/TLSProxy/Certificate.pm file and the function parse. I was seeing the data that is initial parsed at the beginning of the TLS13 if check. 

Example Data I was seeing: 0B 00 03 32 00 02 00 00 C7 00 02 6B

This would be encrypted data and points to the Handshake Header. 0B -- Certificate 00 03 32 -- 818 bytes 00 02 -- Handshake message sequence number of 2, 00 00 C7 -- Fragment offset of 199, 00 02 6B - 619 bytes length of this fragment.

Reply:
Makes sense, then please use: DTLS_MESSAGE_HEADER_LENGTH which is the DTLS Handshake Message Header length.